modules/op-chains-signing-secrets-ecdsa.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-signing-secrets-ecdsa_{context}"]
+= Generating the x509 key pair
+
+You can generate the `x509` key pair by setting the `generateSigningSecret` parameter to `true`. 
+The {pipelines-title} Operator generates an `ecdsa` type key pair: `x509.pem` private key and `x509-pub.pem` public key, and stores it in the `signing-secrets` secret in the `openshift-pipelines` namespace.
+
+The {pipelines-title} Operator *does not* provide the following functions to limit potential security issues:
+
+* Key rotation 
+* Auditing key usage
+* Proper access control to the key
+
+[IMPORTANT]
+====
+Ensure that you store the `x509-pub.pem` public key in a safe place to verify later artifact attestations.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`) utility.
+* You are logged in to your {OCP} cluster with administrative rights for the `tekton-chains` namespace.
```

Comment:
Chains is installed in `openshift-pipelines` namespace right ?

Reply:
+1

Reply:
Hello @PuneetPunamiya Does this mean all of the existing chapters that have the same prerequisite are incorrect? Because I copied it from the others, like this section:  https://docs.openshift.com/pipelines/1.16/secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#creating-mounting-kms-authentication-token-secret_using-tekton-chains-for-openshift-pipelines-supply-chain-security

If that is the case, I will fix the existing prerequisites as well.

Reply:
Yeah could you please update that as well, thanks for the other catch as well 🙌🏻